content/5_Plat/Matrix_Expo.mdx

```diff
@@ -124,7 +128,48 @@ int main() {
 	cout << base[0][1];
 }
 ```
+</CPPSection>
+
+<PySection>
+```python
+import sys
+MOD = 1000000007
+
+def mul(A, B, MOD):
+    C = [[0,0], [0,0]]
+
+    for i in range(2):
+        for j in range(2):
+            for k in range(2):
+                C[i][j] += ((A[i][k] % MOD) * (B[k][j] % MOD)) % MOD
```

Comment:
not necessary for so many mods? just one at the end is fine since the previous values are already modded

Reply:
^^

Reply:
made the changes